packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -200,7 +200,7 @@ abstract contract FundingRateApplier is FeePayer {
         return OptimisticOracleInterface(finder.getImplementationAddress(OracleInterfaces.OptimisticOracle));
     }
 
-    function _getConfig() internal view returns (ConfigStoreInterface.ConfigSettings memory) {
+    function _getConfig() internal returns (ConfigStoreInterface.ConfigSettings memory) {
```

Comment:
why is this no longer view? is this not separate to the point of this issue?

Reply:
It's no longer view because this method calls `getCurrentConfig()` which is not a view method anymore